### PR TITLE
Use first mealtime when throwing error text

### PR DESCRIPTION
**Pull request by ryanmitchell**
@sampoyigi are you happy with this or should this error be updated to show all mealtimes?

### CODE REVIEW

**Comment by sampoyigi** [comment]
I'd show all mealtimes so it's clear to the customer when next the item will be available.

**Comment by ryanmitchell** [comment]
Something like this?

**Comment by sampoyigi** [comment]
Good but would result in multiple `Item A is available for Dinner ONLY between (%s - %s)`, instead we want `Item A is available for Breakfast  (%s - %s), Dinner (%s - %s)`

**Comment by ryanmitchell** [comment]
Ok - so that would mean 2 strings in the lang file, one for "item A is available for", and then another for "Breakfast (%s - %s)" which is then repeated? Or do you know of a better way to handle this?

**Comment by sampoyigi** [comment]
Yep thats the way to go

**Comment by ryanmitchell** [comment]
Cool, updated

File: language/en/default.php [hunk 1]
```diff
@@ -96,7 +96,8 @@
     'alert_coupon_not_applied' => 'Your coupon can not be applied on orders below %s.',
     'alert_coupon_maximum_reached' => 'Maximum number of redemption for the coupon has been reached.',
     'alert_coupon_order_restriction' => 'Your coupon can not be applied to <b>%s</b> orders',
-    'alert_menu_not_within_mealtime' => '%s is available for %s ONLY between (%s - %s)',
+    'alert_menu_not_within_mealtime' => '%s is available for %s',
```
Comment (sampoyigi) [inline language/en/default.php:99]: Would rename this to `alert_menu_not_within_mealtimes` so it doesn't conflict with stored lang strings causing errors with `sprintf`

**Review by sampoyigi — CHANGES_REQUESTED** [review]
See comments

File: language/en/default.php (hunk 1 above)
Comment (ryanmitchell) [reply language/en/default.php]: No problem - updates

**Pull request merged**